app/views/partials/droits-eligibles-list.html

```diff
@@ -1,96 +1,92 @@
-<uib-accordion close-others="false">
-  <uib-accordion-group is-open="droit.open"
-     class="result {{ getAccordionClass(droit) }}"
-     ng-repeat="(droitId, droit) in list|orderObjectBy:'label'">
-    <uib-accordion-heading>
-      <div class="row">
-        <div class="col-xs-2 col-sm-1">
-          <img class="droit-logo"
-               ng-src="/img/{{ droit.imgSrc }}"
-               alt="">
+<table class="table table-result">
```

Comment:
Because http://getbootstrap.com/css/#tables :(
I just used it because we use bootstrap anyway, it sets a few useful rules.